include/bluetooth/mesh/light_ctrl_srv.h

```diff
@@ -200,7 +200,7 @@ int bt_mesh_light_ctrl_srv_on(struct bt_mesh_light_ctrl_srv *srv);
  *  state). Calling this function temporarily disables occupancy sensor
  *  triggering (referred to as "manual mode" in the documentation). The server
  *  will remain in manual mode until the manual mode timer expires, see
- *  @ref CONFIG_BT_MESH_LIGHT_CTRL_SRV_TIME_MANUAL.
+ *  @em CONFIG_BT_MESH_LIGHT_CTRL_SRV_TIME_MANUAL.
```

Comment:
Would this work?
:option:`CONFIG_BT_MESH_LIGHT_CTRL_SRV_TIME_MANUAL`

Reply:
I'm afraid not, this is syntax used with Sphinx/Rst, not doxygen.

Reply:
We could add an alias to fix this:
```
"option{1}=<a href=\"@KCONFIG_OUTPUT@/\1.html\">\1</a>"
```
then use it in the headers like this:
```
@option{CONFIG_2ND_LEVEL_INTERRUPTS}
```

Reply:
This is quite clever, and it did work on my machine, but unfortunately it does use absolute file paths not URLs, which would not allow it to work on the server when published. For intersphinx we generate the appropriate relative paths with some python calls so something similar can be tried here...